index.js

```diff
@@ -85,13 +90,36 @@ exports.getToday = function (_date, pretty) {
   return pretty ? prettyFormat(mish) : mish;
 };
 
+var constrainListToSeder = function(list, thisSeder){
+  if(!seder[thisSeder]){
+    return list;
+  }
+
+  var allowedMasechtos = _.map(seder[thisSeder], function(masechtaName){ return _.indexOf(names, masechtaName) });
+
+  var partOfSeder = function(dailyQuota) { return _.contains(allowedMasechtos,  dailyQuota.t); }
+  list = _.chain(list)
+    .filter(function(dailyQuota){
+      return _.some(dailyQuota, partOfSeder);
+    }).map(function(dailyQuota){
+      return _.filter(dailyQuota, partOfSeder);
+    }).value();
+
+  return list;
+}
+
 exports.buildCalendar = function (o) {
   mas_index = 0;
   perek_index = 0;
   mish_index = 0;
 
   var day = new moment(o.start || moment([2010, 6, 4]));
   var list = buildList(o.per_day || 2, o.perakim);
+
+  if(o.seder){
+    list = constrainListToSeder(list, o.seder);
```

Comment:
Also you might want to do the limiting in the buildList function (more efficient).